### PR TITLE
Added django-polaris package path for local development without Docker. 

### PR DESCRIPTION
**Pull request by lijamie98**

### CODE REVIEW

File: example/Pipfile [hunk 1]
```diff
@@ -9,6 +9,11 @@ watchdog = "<0.10"
 [packages]
 gunicorn = "*"
 psycopg2-binary = ">=2.9, <3.0"
+
+# Uncomment the following line if you would like to run the dev without docker
```
Comment (JakeUrban) [inline example/Pipfile:13]:
```suggestion
# Uncomment the following line if you would like to run the example without docker
```

File: example/Pipfile [hunk 2]
```diff
@@ -9,6 +9,11 @@ watchdog = "<0.10"
...
+# django-polaris = {path = "..", editable = true}
+
+# Comment the following line if you would like to run the dev without docker
```
Comment (JakeUrban) [inline example/Pipfile:16]:
```suggestion
# Comment the following line if you would like to run the example without docker
```

File: example/Pipfile [hunk 3]
```diff
@@ -9,6 +9,11 @@ watchdog = "<0.10"
 [packages]
 gunicorn = "*"
 psycopg2-binary = ">=2.9, <3.0"
+
+# Uncomment the following line if you would like to run the dev without docker
+# django-polaris = {path = "..", editable = true}
```
Comment (JakeUrban) [inline example/Pipfile:14]: Is `..` correct? Is this path relative to the current working directory? For example, if I run from the `example` directory does the path need to be different than if I run from the `django-polaris` directory?

**Review by JakeUrban — APPROVED** [review]
A couple suggestions and one question. But if they're all resolved then it LGTM

File: example/Pipfile (hunk 1 above)
Comment (lijamie98) [reply example/Pipfile]: Will change

File: example/Pipfile (hunk 2 above)
Comment (lijamie98) [reply example/Pipfile]: Will change

File: example/Pipfile (hunk 3 above)
Comment (lijamie98) [reply example/Pipfile:14]: Cannot find a way to inject a path with something like $HOME/django-polaris. 

The `pipenv install --dev` must be run from the example folder. This issue can be taken care of in the document.

**Pull request merged**